dist/lang/es.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ForienUnidentifiedItems": {
+  "forien-unidentified-items": {
```

Comment:
Why the change to the module's localization key?

Reply:
A "standard" that I have seen in other modules, that of using the module id as the primary key of the language files and that I also use, but it is a preference at the development level, I do not see particular problems, but I can restore the old label if you prefer it

Reply:
No, this is fine. I was just curious.